core/ecschema-metadata/src/Metadata/RelationshipClass.ts

```diff
@@ -371,6 +371,13 @@ export class RelationshipConstraint implements CustomAttributeContainerProps {
     return false;
   }
 
+  public static isRelationshipConstraint(object: any): object is RelationshipConstraint {

```

Comment:
Can we make all of the `isXX` method a property getter instead of static methods?  This would more closely match what is done for `IModelDb` and `IModelConnection`